### ISSUE TITLE
CMS, components, design system, sitewide 508-defect-2 [KEYBOARD, AXE-CORE, SEMANTIC MARKUP]: Landmarks MUST have a unique role or role/label/title (i.e. accessible name) combination; main nav and sidebar nav conflict

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#6368]
#  [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

```diff
! Team: VSA Public Websites, AMA updates
```

Please see issue https://github.com/department-of-veterans-affairs/va.gov-team/issues/7723 — this is the same issue, but in a different project.

> **Please note:** This particular ticket is an issue with the CMS templates, and also exists throughout our React apps. The main navigation, side navigation, and breadcrumbs need distinct `aria-labels` to distinguish them for assistive technology. Ideally, this would also be fixed in our React apps.
> * main nav: aria-label="main"
> * side nav: aria-label="secondary"
> * breadcrumbs: aria-label="breadcrumb"

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

Currently the VA.gov main nav, breadcrumbs, and the sidebar nav use the `<nav>` element. Each landmark **MUST** have a unique role or accessible name (i.e. role, label, title) combination. Sighted keyboard and screen reader users navigate pages by landmarks and need to be able to distinguish between them. If landmark roles are repeated on a page, then they need to be labelled uniquely.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a keyboard user, I want to navigate the page by landmarks and distinguish between each landmark.

## Environment

* Operating System: all
* Browser: all
* Screen reader: any
* Server destination: production

## Steps to Recreate

1. Enter `https://www.va.gov/disability/how-to-file-claim/` or `https://www.va.gov/disability/after-you-file-claim/` in browser
2. Start screenreading device of choice
3. Navigate the page by landmarks (see vendor guidance below)
4. Alternatively, scan the page with axe or axe-coconut to verify the issue

## Possible Fixes (optional)

There are three navigation items on this page:
- sitewide main vetnav megamenu
- breadcrumbs component
- sidebar nav

If a page includes more than one navigation landmark, each should have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable

```diff
! <!-- current -->
! <div id="vetnav" role="navigation">
! <nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs" id="va-breadcrumbs">
! <nav id="va-detailpage-sidebar" data-drupal-sidebar="true" class="va-drupal-sidebarnav usa-width-one-fourth va-sidebarnav">

<!-- possible fix -->
<div id="vetnav" role="navigation" aria-label="main">
<nav aria-label="breadcrumb" aria-live="polite" class="va-nav-breadcrumbs" id="va-breadcrumbs">
<nav id="va-detailpage-sidebar" data-drupal-sidebar="true" class="va-drupal-sidebarnav usa-width-one-fourth va-sidebarnav" aria-label="secondary">
```

## WCAG or Vendor Guidance (optional)

* [W3C ARIA Navigation Landmark Examples](https://www.w3.org/TR/2017/NOTE-wai-aria-practices-1.1-20171214/examples/landmarks/navigation.html)
* [axe-core 3.4 - Landmarks must have a unique role or role/label/title (i.e. accessible name) combination](https://dequeuniversity.com/rules/axe/3.4/landmark-unique?application=AxeChrome)
* [Apple VoiceOver - Navigate webpages using landmarks](https://help.apple.com/voiceover/info/guide/10.11/en.lproj/vo35709.html)

**Comment by jenstrickland** [comment]
@1Copenut - I assigned you because I'm not sure who to assign this to. This appears to be a CMS issue.

**Comment by MickinSahni** [comment]
Same with this one, Trevor, should this CMS item be transferred to you? @1Copenut

**Comment by 1Copenut** [comment]
@MickinSahni Yes, I'll take this one, but I'm working on some automation efforts this VSP sprint, started 4/2.

**Comment by jenstrickland** [comment]
Added to the log of sitewide issues, noted that this work includes the issue, and removing the team from the labels.

**Comment by joshkimux** [comment]
@laflannery tagging you in here just in case!

**Comment by jaredcunha** [comment]
An accessible name has been given to the breadcrumb component.

**Comment by laflannery** [comment]
Thanks for the update @jaredcunha. Is there something that individual teams will need to do to add this label to existing instances of the component? Or will it get added automatically? I'm looking at [the location finder](https://www.va.gov/find-locations) for example

**Comment by jaredcunha** [comment]
@laflannery Were you talking about the instance of the breadcrumbs on that page? Breadcrumbs and pagination now have accessible names.

<img width="793" alt="Screenshot 2023-06-06 at 12 20 20 PM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/1094951/2e282ef4-2d2c-41bc-9586-754560123af4">

**Comment by laflannery** [comment]
Closing in favor of more targeted issues:
- [DST issue](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1999) to add label to sidenav
- [PW specific issue](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14870) to add label to sidenav because there is not yet a component
- The main nav will be address during the header redesign project currently in progress
